### PR TITLE
New bee related items

### PR DESCRIPTION
**Pull request by TDcoolguy300**
A new spell for the wizard, Dr. BEES costume for the costume vending
machine, two new giant bee mobs, new drink that involves Honey and Gin.
Also fixes Conjuration sounds, adds in a new sound randomization system that
can be applied to certain spells, and the ability to add sounds to
previously silent spells.

### CODE REVIEW

**Comment by TDcoolguy300** [comment]
And i forgot to change the title to "This is just the BEEgining!"

**Comment by Kokojos** [comment]
Ayyy this seems good.

Waiting on another +1 or two since this isin't a bugfix.

**Comment by Nokington** [comment]
This needs to be talked about among the heads first.

**Comment by ghost** [comment]
SNOWFLAKE CODE DETECTED :< YH U NO MAKE PARENT PROC WITH THE STUFF FOR SOUNDS.

**Comment by TDcoolguy300** [comment]
Actually, your right on that part, i will not leave out the Random sound out of the next change, randomization code is still being worked on.

**Comment by TDcoolguy300** [comment]
Right, here's a Conundrum for you Erik, Although the parent file Spell will take the cast_sound var, it will not execute the sounds on cast() due to the separation of cast() between the base magics, such as Conjouration. TL:DR /obj/effect/proc_holder/spell/proc/cast(list/targets) != /obj/effect/proc_holder/spell/aoe_turf/conjure/cast(list/targets)

**Comment by Chnkr** [comment]
Can't you just create `/obj/effect/proc_holder/spell` and add the parent calls in there, and call parent?

**Comment by ghost** [comment]
two small things here.
- Please stop using the workinprogress folder, and put it into the codebase.
- The sounds are still snowflaky, why not rip https://github.com/tgstation/-tg-station/pull/9447 and use that? that works, and isn't snowflaky :+1:

File: code/datums/spell.dm [hunk 1]
```diff
@@ -209,7 +210,7 @@ var/list/spells = typesof(/obj/effect/proc_holder/spell) //needed for the badmin
 				smoke.start()
 
 
-/obj/effect/proc_holder/spell/proc/cast(list/targets)
+/obj/effect/proc_holder/spell/proc/cast(list/targets) // Wont work in other magic anyhow
```
Comment (ghost) [inline code/datums/spell.dm]: this isn't needed.

File: code/modules/reagents/Chemistry-Reagents/Consumable-Reagents/Food-Reagents.dm [hunk 2]
```diff
@@ -434,4 +434,10 @@ datum/reagent/consumable/cherryjelly
 	description = "Totally the best. Only to be spread on foods with excellent lateral symmetry."
 	color = "#801E28" // rgb: 128, 30, 40
 
+datum/reagent/consumable/honey
+	name = "Honey"
+	id = "honey"
+	description = "Some gooey honey, nothing really special about this"
+	color = "##a98307" // rgb: 169, 131, 7
```
Comment (ghost) [inline code/modules/reagents/Chemistry-Reagents/Consumable-Reagents/Food-Reagents.dm]: double `##`?

Comment (TDcoolguy300) [inline code/modules/reagents/Chemistry-Reagents/Consumable-Reagents/Food-Reagents.dm]: Weird, didnt seem to cause an issue, i will pull that extra # out in a bundle

File: code/datums/spells/conjure.dm [hunk 3]
```diff
@@ -13,16 +13,57 @@
...
+	for(var/i=0,i<summon_amt,i++)
+		if(!targets.len)
+			break
+		var/summoned_object_type = pick(summon_type)
+		var/spawn_place = pick(targets)
+		if(summon_ignore_prev_spawn_points)
+			targets -= spawn_place
+		if(ispath(summoned_object_type,/turf))
+			if(istype(get_turf(usr),/turf/simulated/shuttle))
+				usr << "<span class='danger'>You can't build things on shuttles!</span>"
```
Comment (ghost) [inline code/datums/spells/conjure.dm:30]: isn't this message a bit odd when using magic?

Comment (TDcoolguy300) [inline code/datums/spells/conjure.dm:30]: That is referring to the Conjuration for Artifactors for Cult, which can build Walls, R-Walls, and flooring on majority of turf. Without that check they can also build on the shuttle turfs. I'd say leave it

Comment (ghost) [inline code/datums/spells/conjure.dm:30]: cult doesn't have any Conjuration for Artifactors, right?

Comment (TDcoolguy300) [inline code/datums/spells/conjure.dm:30]: Yes it does, under construct_spells.dm.

**Comment by ghost** [comment]
Looks fine to me.

**Comment by Nokington** [comment]
Merge errors

**Comment by Nokington** [comment]
Closing for now, gonna try making a PR from my fork which is uptodate

**Pull request closed without merging**